pkgs/cronet_http/README.md

```diff
@@ -40,22 +38,45 @@ void main() async {
   final Client httpClient;
   if (Platform.isAndroid) {
     final engine = CronetEngine.build(
-        cacheMode: CacheMode.memory,
-        cacheMaxSize: 2 * 1024 * 1024,
-        userAgent: 'Book Agent');
+      cacheMode: CacheMode.memory,
+      cacheMaxSize: 2 * 1024 * 1024,
+       userAgent: 'Book Agent',
+    );
     httpClient = CronetClient.fromCronetEngine(engine);
   } else {
     httpClient = IOClient(HttpClient()..userAgent = 'Book Agent');
   }
 
-  final response = await client.get(Uri.https(
+  final response = await client.get(
+    Uri.https(
       'www.googleapis.com',
       '/books/v1/volumes',
-      {'q': 'HTTP', 'maxResults': '40', 'printType': 'books'}));
+      {'q': 'HTTP', 'maxResults': '40', 'printType': 'books'},
+    ),
+  );
 }
 ```
 
+### Use embedded Cronet
+
+The embedded [Cronet][] is provided as `org.chromium.net:cronet-embedded`
+which does not require to use the [Google Play Services][].
+The library uses `dart-define` to switch to the embedded implementation
+when `cronetHttpNoPlay` is set to `true`.
+For example:
+
+```
+flutter run --dart-define=cronetHttpNoPlay=true
```

Comment:
The convention seems to be to use name_with_underscore. So maybe:

cronet_http_no_play=true

Reply:
Any guide suggesting this?

Reply:
No, I just looked at some existing defines e.g. `test_runner.configuration`.

But it is not consistent so feel free to leave it this way.